lib/src/grapheme_clusters/table.dart

```diff
@@ -3,319 +3,384 @@
 // BSD-style license that can be found in the LICENSE file.
 
 // Generated code. Do not edit.
-// Generated from https://www.unicode.org/Public/12.0.0/ucd/auxiliary/GraphemeBreakProperty.txt
-// and https://unicode.org/Public/emoji/12.0/emoji-data.txt.
+// Generated from [https://unicode.org/Public/UCD/latest/ucd/auxiliary/GraphemeBreakProperty.txt](../../third_party/Unicode_Consortium/GraphemeBreakProperty.txt)
```

Comment:
Is there a permanent link without `latest` that we could use? I expect the content behind this link could change.

Reply:
I'm curious - what are the steps to regenerate this? Could be add those to a readme / and or commit a tool for this in `tool/`?

Reply:
I have a set of scripts. They are not cleaned up for publishing - they never left my personal computer where I originally wrote them, but I can probably make them presentable and cut out the parts that are no longer used. (The directory they're in is called `tools`, I guess the recommended name is `tool`.)

I chose to go with `latest` because then I *don't* need to update the URIs next time I want to update.
If the naming system is consistent, I can probably just replace `latest` with `15.0.0` everywhere to get the current version. Maybe I can make the scripts find the version number (it's in the file itself) can find a persistent URI for the file fetched from `latest`.

Re: Changelog. I'm sure I wrote at least one such update today. Probably on another branch ☹️

Reply:
Ah, I don't know how I overlooked that each of these are downloaded too. No need for a permalink if we also are storing the content at time of generation.